core/backend/src/IpcHost.ts

```diff
@@ -224,6 +224,15 @@ class IpcAppHandler extends IpcHandler implements IpcAppFunctions {
   public async openStandalone(filePath: string, openMode: OpenMode, opts?: StandaloneOpenOptions): Promise<IModelConnectionProps> {
     return StandaloneDb.openFile(filePath, openMode, opts).getConnectionProps();
   }
+  public async openSnapshot(filePath: string, opts?: SnapshotOpenOptions | undefined): Promise<IModelConnectionProps> {
```

Comment:
Why is `SnapshotOpenOptions` optional and also a union with `undefined`? Looks like `SnapshotDb.openFile` is just optional.

Reply:
No reason, I'll remove `undefined`.

In the end, `opts` type will still resolve to same type: `SnapshotOpenOptions | undefined`